test/openapi3/general/readOnlyWriteOnly.yaml

```diff
@@ -33,19 +33,19 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/UserOptional'
-  /users/utf8:
+  /users/hal:
     post:
       requestBody:
         required: true
         content:
-          application/json;utf8:
+          application/hal+json:
```

Comment:
Why did you change it to hal?

Reply:
I think that utf8 part should never be part of API spec, instead we should handle requests that include it in header as just application/json
Support for that is implemented here: https://github.com/Zooz/api-schema-builder/pull/29